tests/clients/test_dapr_grpc_client.py

```diff
@@ -150,6 +152,27 @@ def test_publish_error(self):
                 data=111,
             )
 
+    @patch.object(settings, 'DAPR_API_TOKEN', 'test-token')
+    def test_dapr_api_token_insertion(self):
+        dapr = DaprClient(f'localhost:{self.server_port}')
+        resp = dapr.invoke_service(
+            id='targetId',
+            method='bytes',
+            data=b'haha',
+            content_type="text/plain",
+            metadata=(
+                ('key1', 'value1'),
+                ('key2', 'value2'),
+            ),
+        )
+
+        self.assertEqual(b'haha', resp.content)
+        self.assertEqual("text/plain", resp.content_type)
+        self.assertEqual(4, len(resp.headers))
```

Comment:
This should be 3 right? The metadata from request (2) and 1 dapr-api-token.

Reply:
No it is 4. I have seen that it is always 1 more than we expect. `user-agent` is added automatically